### ISSUE TITLE
regenerative materia blob tox damage not healing

### ISSUE THREAD

**Issue by gorillion** [tgstation/tgstation#85360]
Reporting client version: 515.1640

## Issue Summary

fought a regenerative materia blob and was dealt tox damage, as is inevitable. used three different toxin healing medicines (hot seiver, pentetic acid, and musiver/syriniver) over the course of the round, none reduced my toxin damage. ive had this happen before in rounds many months ago, i had assumed chems weren't working, but other chems for healing burns or brute still processed within my body and healed me. toxins seemed to heal for other crewmembers, might be important that i was a plasmaman?

## Round ID:
232803

## Testmerges:
- [Block atmos processing on ReservedTurfs\:tm: ](https://www.github.com/tgstation/tgstation/pull/84873)
- [Spacemove refactor - Newtonian physics](https://www.github.com/tgstation/tgstation/pull/84869)
- [Makes cyborg damage more... showy](https://www.github.com/tgstation/tgstation/pull/85298)

## Reproduction:

get damaged by a regenerative materia blob's toxin as a plasmaman, watch as your toxin is seemingly unremovable

**Comment by necromanceranne** [comment]
Did you inject them all together?

How stable was your liver?

**Comment by gorillion** [comment]
> Did you inject them all together?


no, this was over the course of the whole rest of the round (probably like ~30 minutes of being stuck at 55-56 tox)

> How stable was your liver?

completely stable as far as i know. i kept scanning myself with a health analyzer and it never showed any organ damage.

**Comment by Sadboysuss** [comment]
tested this: plasmamen don't process toxins, neither can they heal toxins, but regenerative materia deals toxins by attacking, including injecting reagents.
